dallinger/recruiters.py

```diff
@@ -397,18 +397,19 @@ def completion_code(self):
     def open_recruitment(self, n: int = 1) -> dict:
         """Create a Study on Prolific."""
 
+        if self.study_domain is None:
```

Comment:
Hi @fmhoeger, it feels like you're forgetting the purpose of `DevProlificRecruiter`? All this logic should go there. Just write an `open_recruitment` method for `DevProlificRecruiter` that prints to the console but doesn't try to contact the Prolific APi.

Reply:
Basically, all I had to do was setting `prolific_workspace` to the value returned by the fake API response.